frontend-react/src/pages/deliveries/Table/DeliveriesTable.test.tsx

```diff
@@ -107,106 +108,168 @@ describe("DeliveriesTable", () => {
     });
 });
 
-describe("DeliveriesTableWithNumberedPagination - with data", () => {
```

Comment:
A lot of this is just reorganizing/scoping tests.  [This one](https://github.com/CDCgov/prime-reportstream/pull/7928/files#diff-27e69cc0f92ac1293b833154b6fe60e745b0bd48c510faecd23188247518eddbR226) is the only net new test here!